includes/Core/Authentication/Authentication.php

```diff
@@ -1171,8 +1182,9 @@ private function get_authentication_oauth_error_notice() {
 					}
 
 					$message = sprintf(
-						/* translators: %s: Error message */
-						esc_html__( 'Site Kit by Google: %s', 'google-site-kit' ),
+						/* translators: %1$s: Plugin name. %2$s: Error message. */
```

Comment:
Maybe we should make this a little more generic to work for both?

E.g.
```php
/* translators: %1$s: Plugin name. %2$s: Message. */
```

This way I think we could use the same comment for notices and errors.

Reply:
Sorted.